### ISSUE TITLE
Bootstrap sliders not working - displaying as block instead of columns

### ISSUE THREAD

**Issue by MartynaC** [bootscore/bootscore#225]
Hello,

I am using Bootscore with Woocommerce theme on my website, I am trying to add a simple product slider based on bootstrap and ACF fields.
I can not make it work, even though it works on my other websites, using different themes. In fact, I have tried many different bootstrap sliders approaches and none of it works – everything gets displayed in one column, underneath each item, I have tried these two codes for example:
https://codepen.io/mephysto/pen/ZYVKRY
https://www.jqueryscript.net/slider/multi-item-carousel-bootstrap.html

and this is the desired code, I have been successfully using on other sites:

<div id="myCarousel" class="carousel slide" data-ride="carousel">
<!– Wrapper for slides –>
<div class="carousel-inner">
<?php
$featured_posts = get_field('wyswietlane_produkty');
$is_active = true; // Only true for the first iteration
$i = 0;

if( $featured_posts ): ?>
<ul class="marka-product-listing">
<div class="row">
<?php foreach( $featured_posts as $post ):

// Setup this post for WP functions (variable must be named $post).
setup_postdata($post); ?>
<?php if ($i % 6 == 0):?>
<div class=" carousel-item item <?php if ($is_active) echo ' active'?>">
<?php endif?>
<li>

<div class="col-md-2 col-6 ">

<a href="<?php the_permalink(); ?>">
<?= get_the_post_thumbnail();?>
</a>
<?php $tags = get_terms( 'product_tag' );
if ($tags) {
foreach( $tags as $tag ) {
echo "<p><span class='text-uppercase new'>" .$tag->name. "</span></p>";
}
}
?>
<h3 class="single-marka-prod"><a href="<?php the_permalink(); ?>"><?php the_title(); ?></a></h3>
<p class="slider-price"><?= wc_get_product()->get_price_html()?></p>

</div>

</li>
<?php if (($i+1) % 6 == 0 || $i == count($featured_posts)-1):?>
</div>
<?php endif?>
<?php
$i++;
if ($is_active) $is_active = false;
endforeach; ?>
</div>
</ul>
<?php
// Reset the global post object so that the rest of the page works correctly.
wp_reset_postdata(); ?>
<?php endif; ?>
</div><!– end item—>
<a class="left carousel-control" href="#myCarousel" data-slide="prev">
<span class="glyphicon glyphicon-chevron-left"></span>
<span class="sr-only">Previous</span>
</a>
<a class="right carousel-control" href="#myCarousel" data-slide="next">
<span class="glyphicon glyphicon-chevron-right"></span>
<span class="sr-only">Next</span>
</a>
</div><!– end of item row–>

</div><!–end of carousel inner–>

Can I get some help with this one, please?
Am I missing something?

**Comment by crftwrk** [comment]
Yes, that‘s because Bootstrap‘s carousel component is not able to show columns by default. Getting your Codepen examples working in a WP loop with Bootstrap carousel is quiet hard.

For this reason we‘ve created the bS Swiper plugin https://bootscore.me/documentation/bs-swiper/ which is able to show WooCommerce products or any other custom post types.

Use this plugin with a simple shortcode and show your customers products in a slideshow.

BTW I converted this issue into a discussion, because this isn‘t a theme bug.

Does this help?
